.clitest/znap.source.find.md

```diff
@@ -0,0 +1,63 @@
+Initialize
+```zsh
+% source .clitest/__init__.zsh
+% .test.setup
+%
+```
+
+Source finds correct plugin file
+```zsh
+% dir=$T_TEMPDIR
+% .test.makeplugin foo.plugin.zsh
+% reply=(); .znap.source.find $dir/foo  #=> --exit 0
+% echo ${reply[-1]:t}
```

Comment:
Use `print -r -- $reply[-1]:t`. It's safer (ignores escape codes and doesn't interpret `-` as starting a flag) and more concise.